src/Arcus.Scripting.LogicApps/Scripts/Cancel-AzLogicAppRuns.ps1

```diff
@@ -1,19 +1,82 @@
 param(
     [Parameter(Mandatory = $true)][string] $ResourceGroupName = $(throw "Name of the resource group is required"),
     [Parameter(Mandatory = $true)][string] $LogicAppName = $(throw "Name of the logic app is required"),
+    [Parameter(Mandatory = $false)][string] $WorkflowName = "",
+    [Parameter(Mandatory = $false)][string] $EnvironmentName = "AzureCloud",
     [Parameter(Mandatory = $false)][int] $MaximumFollowNextPageLink = 10
 )
 
+$Global:accessToken = "";
+$Global:subscriptionId = "";
+
 try {
-    $runs = Get-AzLogicAppRunHistory -ResourceGroupName $ResourceGroupName -Name $LogicAppName -FollowNextPageLink -MaximumFollowNextPageLink $MaximumFollowNextPageLink | 
-        Where-Object {$_.Status -eq 'Running'}
+    if ($WorkflowName -eq "") {
+        $runs = Get-AzLogicAppRunHistory -ResourceGroupName $ResourceGroupName -Name $LogicAppName -FollowNextPageLink -MaximumFollowNextPageLink $MaximumFollowNextPageLink | 
+            Where-Object {$_.Status -eq 'Running'}
 
-    foreach ($run in $runs) {
-        Stop-AzLogicAppRun -ResourceGroupName $ResourceGroupName -Name $LogicAppName -RunName $run.Name -Force
-        Write-Verbose "Cancelled run $run.Name for the Azure Logic App '$LogicAppName' in resource group '$ResourceGroupName'"
-    }
+        foreach ($run in $runs) {
+            Stop-AzLogicAppRun -ResourceGroupName $ResourceGroupName -Name $LogicAppName -RunName $run.Name -Force
+            Write-Verbose "Cancelled run $run.Name for the Azure Logic App '$LogicAppName' in resource group '$ResourceGroupName'"
```

Comment:
I think that we need to use `$($run.Name)` to concatenate a property directly in a `string`, no?

Reply:
aligned the logging in https://github.com/arcus-azure/arcus.scripting/pull/415/commits/f901fcd105ec31ffd9944f338e7e9ba730abbcdb